enhancements/hypershift/topology-aware-scheduling/topology-aware-scheduling.md

```diff
@@ -0,0 +1,203 @@
+---
+title: topology-aware-scheduling
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@ffromani"
+  - "@csrwng"
+  - "@sjenning"
+approvers: 
+  - "@jparrill"
+  - "@alvaroaleman"
+  - "@csrwng"
+  - "@sjenning"
+api-approvers:
+  - None
+creation-date: 2024-08-29
+last-updated: 2024-08-29
+tracking-link: 
+  - https://issues.redhat.com/browse/CNF-11154
+see-also:
+  - https://github.com/openshift-kni/numaresources-operator/tree/main
+---
+
+# NumaResources-Operator Deployment on a Hosted Clusters
+
+## Summary
+
+Topology-Aware Scheduling (TAS) enhances Kubernetes/OpenShift (K8S/OCP) scheduler 
+by adding NUMA awareness, enabling more granular scheduling decisions 
+with respect to NUMA resource locality.
+The Numa-Resources-Operator (NROP) is responsible for deploying and enabling
+TAS capabilities on the cluster.
+This proposal aims to address the gaps identified
+during the installation of NROP on the HyperShift (HCP) platform.
+
+## Motivation
+
+See the HCP (HyperShift) project goals.
+These goals should not come at the cost of giving up any existing OCP (OpenShift) features.
+On standard OCP it is possible to deploy NROP (Numa-Resources-Operator) as a day-2 operator 
+to optimize scheduling decisions based on the node's hardware layout.
+It's a desire to have the same functionality on HyperShift as well.
+
+
+### User Stories
+
+* As a cluster service consumer, I want to be able to use TAS capabilities on HyperShift's hosted clusters in a similar fashion to standard OCP clusters.
+
+* As an administrator, I want to be able to install NROP on HyperShift's hosted clusters in a similar fashion to standard OCP clusters.
+
+### Goals
+
+Allow a day-2 NROP deployment on a hosted cluster 
+to support TAS (Topology-Aware-Scheduling) over HyperShift platform.
+
+### Non-Goals
+
+- N/A
+
+## Proposal
+
+NROP is a day-2, optional operator that users can install on demand.
+To make the procedure as similar as possible to standard OCP clusters,
+it is suggested to install the operator and its managed components on the hosted-cluster.
+The following paragraphs detail the identified gaps and suggest resolution for each. 
+
+### Architectural Overview
+
+TAS solution comprises three main components:
+
+Resource-Topology-Exporter (RTE) - Responsible for collecting node information (mostly NUMA resources allocations) 
+and advertises it as a CR object named NRT (Node-Resource-Topology)
+
+Secondary-Scheduler - An add-on to the default scheduler which is responsible for
+scheduling the pod in the most suitable node according to the data exposed by the NRT object.
```

Comment:
probably nitpicking, but this is not really an addon. Meaning, it's fully separated, complementary scheduler which enhances the cluster.

Reply:
I Rewrite the sentence thanks!